src/test/java/core/basesyntax/service/WriterImplTest.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.Before;
+import org.junit.Test;
+
+public class WriterImplTest {
+    private static final String FILE_PATH = "src/main/java/core/basesyntax/db/destination.csv";
+    private static final String APPLE = "apple";
+    private static final String BANANA = "banana";
+    private static final String SIGN_SEPARATOR = ",";
+    private static final int APPLE_COUNT = 5;
+    private static final int BANANA_COUNT = 7;
+    private static final Writer writer = new WriterImpl();
```

Comment:
initialize this service in a method annotated with Before or BeforeClass (depends on the version of junit)